website/docs/reference/warehouse-profiles/firebolt-profile.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Firebolt Profile"
+---
+
+
+:::info Community plugin
+
+Some core functionality may be limited. If you're interested in contributing, check out the source code for each repository listed below.
+
+:::
+
+## Overview of dbt-firebolt
+
+**Maintained by:** Community  
+**Author:** Anders Swanson and Eric Ford
+**Source:** [Github](https://github.com/firebolt-db/dbt-firebolt)  
+**dbt Cloud:** Not Supported  
```

Comment:
When you launch, we would be happy to provide a dbt slack channel if you would like. It would be called #db-firebolt

Reply:
or make us one now, so we can include it in this PR? #lazy

Reply:
That would be great!

Reply:
haha I can do that with the promise that you will maintain it :)

Reply:
Here you go, here's the link 

https://getdbt.slack.com/archives/C02PYT5CXN0

The language is `dbt Slack channel: [Slack](https://getdbt.slack.com/archives/C02PYT5CXN0)`

Reply:
10x!